openmetadata-ui/src/main/resources/ui/src/components/Glossary/GlossaryV1.component.tsx

```diff
@@ -541,6 +547,27 @@ const GlossaryV1 = ({
               </Space>
             )}
           </div>
+          <Space className="m-b-md" data-testid="updated-by" size={8}>
+            <Typography.Text className="text-grey-muted">
+              {t('label.updated-by')} -
+            </Typography.Text>
+            <Space size={8}>
+              <ProfilePicture
+                displayName={selectedData.updatedBy}
+                id={selectedData.id}
+                name={selectedData.updatedBy || ''}
+                textClass="text-xs"
+                width="20"
+              />
+              <Typography.Text>
+                <Link to={getUserPath(selectedData.updatedBy ?? '')}>
+                  {selectedData.updatedBy}
+                </Link>
+                {` ${t('label.on-lowercase')} 
+               ${formatDateTime(selectedData.updatedAt || 0)}`}
+              </Typography.Text>
+            </Space>
+          </Space>
```

Comment:
If updatedBy is not present then we should not render it.

Reply:
it will always present as someone will create/update it